### PR TITLE
Update more docs to have Goerli

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: ENG-2449, DOCS-622, ENG-2484

### CODE REVIEW

File: src/docs/developers/build/basic-contract.md [hunk 1]
```diff
@@ -25,9 +25,8 @@ Accept the popup that shows up in MetaMask and you'll be connected to the Optimi
 ## Getting some testnet ETH
 
 Like on Ethereum, transaction fees on Optimism are paid in ETH.
-You'll need some ETH on the Optimism Kovan testnet to deploy a contract.
-The [Paradigm Multifaucet](https://faucet.paradigm.xyz/) is a quick and easy way to get ETH on a whole bunch of testnets, including the Optimism Kovan testnet.
-You'll also be sent some other tokens and assets that you can use to play around even more once you're done with this guide.
+You'll need some ETH on the Optimism Goerli testnet to deploy a contract.
+[See here for directions on how get that ETH](../../useful-tools/networks/#test-eth).
```
Comment (Nicca42) [inline src/docs/developers/build/basic-contract.md:29]: See how to get testnet ETH [here]()

File: src/docs/guides/cex-dev.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ To get the total SNX balance of a user that uses Optimism you need to:
 1. Connect to a standard Ethereum endpoint and send a `balanceOf` query to address `0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f`.
 1. Connect to an Optimism endpoint and send a `balanceOf` query to address `0x8700daec35af8ff88c16bdf0418774cb3d7599b4`.
 
+<!--
+GOON: Add a Georli example when we have a token listing with that chainId
```
Comment (Nicca42) [inline src/docs/guides/cex-dev.md:38]: Can we make this a TODO rather than what ever a GOON is?

File: src/docs/guides/wallet-dev.md [hunk 3]
```diff
@@ -31,6 +31,11 @@ For example, looking at the **SNX** token, the [Optimism token list](https://sta
 | 42 | Kovan (test network) | `0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f` |
 | 69 | Optimistic Kovan (test network) | `0x0064A673267696049938AA47595dD0B3C2e705A1` |
 
+
+<!--
+GOON: Add a Georli example when we have a token listing with that chainId
```
Comment (Nicca42) [inline src/docs/guides/wallet-dev.md:36]: TODO

File: src/docs/useful-tools/explorers.md [hunk 4]
```diff
@@ -20,9 +28,10 @@ It's also got some Optimism-specific features:
 - [A tool for finalizing L2-to-L1 transactions](https://optimistic.etherscan.io/messagerelayer)
 - And more! Just check it out and click around to find all of the available features.
 
-Etherscan currently provides the most advanced block explorers for all Optimism Ethereum networks.
-
-
+::: tip Testnet migration
+We expect to have Etherscan for the Kovan testnet until August 1st, 2022.
+A few weeks after that it will be available for the new Goerli testnet.
```
Comment (Nicca42) [inline src/docs/useful-tools/explorers.md:33]: In the transition period you can use Blockscout

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/guides/cex-dev.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/guides/cex-dev.md:38]: GO ON, and sure.

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**